docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,194 @@
+* Add lineage support for output columns in `SELECT` queries. ({issue}`26241`)
+* Require JDK 25 to build and run Trino. ({issue}`27153`)
+* Require the JDK Vector API to be enabled at runtime. ({issue}`27340`)
+* Add support for setting and dropping column defaults via ALTER TABLE ... ALTER COLUMN" at current. ({issue}`26162`)
+* Add {func}`array_first` and {func}`array_last` functions. ({issue}`27295`)
+* {{breaking}} The configuration property `task.statistics-cpu-timer-enabled` is now defunct and must be removed. ({issue}`27504`)
+* Allow field name declaration in ROW literal.  For example, `row(1 as a, 2 as b)` is now legal. ({issue}`25261`)
+* Deprecate `EXPLAIN (TYPE LOGICAL)`. `EXPLAIN (TYPE DISTRIBUTED)` should be used instead. ({issue}`27434`)
+* Remove `prefer_streaming_operators` session property. `SET SESSION task_concurrency=1` can be used to achieve the same behaviour as `prefer_streaming_operators`. ({issue}`27506`)
+* Improve performance of remote data exchanges on CPUs supporting the required SIMD extensions. This can be disabled by setting `exchange.experimental.vectorized-serde.enabled=false`. ({issue}`27426`, {issue}`26919`)
+* Enhance after `RemoveTrivialFilters` call `RemoveEmptyUnionBranches` to prune empty union branches. ({issue}`21506`)
+* Improve performance of {func}`array_sort` function. ({issue}`27272`)
+* Improve performance of {func}`repeat` function. ({issue}`27369`)
+* Improve performance of data exchanges involving variable width data. ({issue}`27377`)
+* Fix planning failure of certain queries where part of the plan is optimized to empty values. ({issue}`21506`)
+* Fix configuring partitioned layout for spooling protocol. ({issue}`27247`)
+* Fix `EXPLAIN (TYPE IO)` failure when query constraint contains type which cannot be cast to `varchar`. ({issue}`27433`)
+
+## Security
+
+## Web UI
+
+* Fix preview UI to render single line queries. ({issue}`27328`)
+* Fix preview UI to render catalog properties correctly. ({issue}`27327`)
+
+## JDBC driver
+
+* Add a `--extra-header` option to support sending arbitrary HTTP headers. ({issue}`15826`)
+
+## Docker image
+
+* Update JDK to 25.0.1. ({issue}`27117`)
+
+## CLI
+
+* Add a `--extra-header` option to support sending arbitrary HTTP headers. ({issue}`15826`)
+* Fix TLS connection failures when using unqualified (single-label, without ".") hostnames. Default Java TLS does not send Server Name Indication (SNI) for such hostnames, which could cause certificate mismatches during TLS termination. ({issue} `27478`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## Delta Lake connector
+
+* Add `APPLICATION_DEFAULT` authentication type for GCS. ({issue}`26984`)
+* Remove unauthentication access support in `SERVICE_ACCOUNT`. ({issue}`26984`)
```

Comment:
Did you mean "unauthenticated access"?

Rephrase as:
```
* Remove support for unauthenticated access when GCS authentication type is set to `SERVICE_ACCOUNT`.
```

Also, mark it as `{breaking}`

Reply:
Yes, meant "unauthenticated access".

updated